samples/samples/pg_snippets.py

```diff
@@ -1330,27 +1321,133 @@ def query_data_with_query_options(instance_id, database_id):
 
     with database.snapshot() as snapshot:
         results = snapshot.execute_sql(
-          "SELECT VenueId, VenueName, LastUpdateTime FROM Venues",
-          query_options={
-            "optimizer_version": "1",
-            "optimizer_statistics_package": "latest",
-          },
+            "SELECT VenueId, VenueName, LastUpdateTime FROM Venues",
+            query_options={
+                "optimizer_version": "1",
+                "optimizer_statistics_package": "latest",
+            },
         )
 
         for row in results:
             print("VenueId: {}, VenueName: {}, LastUpdateTime: {}".format(*row))
     # [END spanner_postgresql_query_with_query_options]
 
 
+# [START spanner_postgresql_jsonb_add_column]
+def add_jsonb_column(instance_id, database_id):
+    """Adds a new JSONB column to the Venues table in the example database."""
+    spanner_client = spanner.Client()
```

Comment:
Please include the following comment on client initializations: https://googlecloudplatform.github.io/samples-style-guide/#clients